mocha.js

```diff
@@ -1999,10 +1999,10 @@ var isatty = tty.isatty(1) && tty.isatty(2);
 exports = module.exports = Base;
 
 /**
- * Enable coloring by default.
+ * Enable coloring by default, except in the browser interface.
  */
 
-exports.useColors = supportsColor || (process.env.MOCHA_COLORS !== undefined);
+exports.useColors = process.env ? (supportsColor || (process.env.MOCHA_COLORS !== undefined)) : false;
```

Comment:
this logic could also be done as 

`process.env && (supportsColor || (process.env.MOCHA_COLORS !== undefined)`